Flask server

dlahtine opened the pull request:
Renamed as requested. Flask is operating as expected.

dlahtine committed:
created basic flask server

dlahtine committed:
Moved flask application to top level, moved src to web.

trevorforrey reviewed (dismissed):
Would you be able to move your python folder outside of the src folder? We will need to discuss file structure and deployment in the future, but in the meantime, anything within the src folder will run through our build tool as we do frontend dev.

dlahtine committed:
Merge branch 'dev' of https://github.com/zawata/S19-Team2 into flask_server

dlahtine committed:
Created endpoints for sun, moon, spacecraft position and orientation.

Created endpoints for accessing and replacing SPK file.

implemented functionality for serving webpages.

dlahtine committed:
Merge branches 'dev' and 'flask_server' of https://github.com/zawata/S19-Team2 into flask_server

dlahtine committed:
Moved flask application to top level, moved src to web.

dlahtine committed:
Task 34: Added file upload functionality, loads spyce (when built beforehand) removed extraneous functions, created file upload endpoint

dlahtine committed:
Merged dev into flask_server

dlahtine committed:
to be under web instead of src

dlahtine committed:
added /data/trajectory

dlahtine committed:
Rearranged items

dlahtine committed:
Changed "spk" to "bsp", the actual format we'll be using
